docker/neumann/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM ubuntu:20.04
+WORKDIR /app
+ADD ./target/release/neumann-collator .
+
+ENTRYPOINT ["./neumann-collator", "-d", "data/node"]
```

Comment:
```
./target/release/neumann-collator --collator --alice --force-authoring --tmp --port 40335 --ws-port 9946 -- --execution wasm --chain resources/rococo-local.json --port 30335
```

See README: https://github.com/OAK-Foundation/OAK-blockchain#launch-the-parachain